openmdao/drivers/tests/test_genetic_algorithm_driver.py

```diff
@@ -153,13 +153,44 @@ def compute(self, inputs, outputs):
         prob['area3'] = 0.0005
         prob.run_driver()
 
-        # Note, GA doesn't do so well with the continunous vars, naturally, so we reduce the space
-        # as much as we can. Objective is stll rather random, but it is close. GA does a great job
+        # Note, GA doesn't do so well with the continuous vars, naturally, so we reduce the space
+        # as much as we can. Objective is still rather random, but it is close. GA does a great job
         # of picking the correct values for the integer desvars though.
         self.assertLess(prob['mass'], 6.0)
         assert_rel_error(self, prob['mat1'], 3, 1e-5)
         assert_rel_error(self, prob['mat2'], 3, 1e-5)
-        #Material 3 can be anything
+        # Material 3 can be anything
+
+    def test_analysis_error(self):
+        np.random.seed(1)
+
+        class ValueErrorComp(ExplicitComponent):
+            """
+            Raises value error. AnalysisError should return the original error.
```

Comment:
This comment isn't quite right.  A solver can raise an AnalysisError if it fails to converge, or certain components will generate AnalysisErrors, but in the case of ValueErrorComp, it will simply raise its ValueError and no AnalysisError will ever be involved.  I would just remove this comment since it will probably cause confusion.